src/ui/popup/main-page/index.tsx

```diff
@@ -5,6 +5,7 @@ import AppSwitch from './app-switch';
 import HelpGroup from './help';
 import SiteToggleGroup from './site-toggle';
 import ThemeGroup from './theme-group';
+import {getLocalMessage} from 'utils/locales';
```

Comment:
`rollup-plugin-typescript2` The compiler for typescript request exact path for imports so this should become something like `import {getLocalMessage} from '../../../utils/locales';`

Reply:
👍